src/app/login.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Layout from './layout';
+
+class UserLogin extends React.Component {
+  getInitialState() {
```

Comment:
FWIW, `getInitialState` won't work with ES6 React components. You'll want to do something like:

```javascript
constructor() {
  this.state = { ... };
}
```